app/controllers/concerns/scholars_archive/works_controller_behavior.rb

```diff
@@ -12,8 +12,32 @@ def edit
       super
```

Comment:
Would you want to fetch the appropriate field & value from the database during `#edit` as well?